src/features/authentication/sign-up/SignUp.service.ts

```diff
@@ -14,7 +15,45 @@ export class SignUpService {
     if (await this.signUpDao.userExists(member)) {
       throw new HttpError(ResponseConflict.message, ResponseConflict.code)
     }
-
+    if (member.password) {
+      member.password = await BcryptUtil.encodeString(member.password);
```

Comment:
You are already using encode method in `Member` entity. Why you are using it in service class.